docs/geedocs/docsrc/answer/geeServerConfigAndSecurity/configureGeeForLDAP.rst

```diff
@@ -22,11 +22,10 @@ Configure GEE Server Admin page for LDAP authentication
 
       .. tip::
 
-         This article explains how to configure GEE Server admin page for LDAP
+         This article explains how to configure GEE Server admin pages for LDAP
          authentication.
 
-      .. rubric:: To configure GEE server admin page for LDAP
-         authentication:
+      .. rubric:: To configure GEE Server admin pages for LDAP authentication:
```

Comment:
I can't add a comment where it belongs, but the capping of LDAP/ldap modules should be the same in #1 and #2 below.